beacon_node/http_api/tests/tests.rs

```
@@ -1956,7 +1956,7 @@ impl ApiTester {
 
         match self
             .client
-            .get_blobs::<E>(CoreBlockId::Slot(test_slot), None, &self.chain.spec)
+            .get_blob_sidecars::<E>(CoreBlockId::Slot(test_slot), None, &self.chain.spec)
```

Can we add some tests for `get_blobs`, some scenarios:
- pre deneb, a fullnode should return blobs
- from fulu, only supernode returns blobs
- from fulu, a fullnode should return an error

---

Added tests for: 
- pre Fulu, a fullnode should return blobs in `get_blob_sidecars()` - calling the `test_get_blobs` function
- post Fulu, a supernode is able to return blobs in `get_blobs_post_fulu_supernode()` - calling the `test_get_blobs` function
- post Fulu, a fullnode will error when asked for blobs in `get_blobs_post_fulu_full_node()`